Dockerfile.tests

```diff
@@ -0,0 +1,9 @@
+FROM nbviewer_nbviewer
```

Comment:
Instead of relying on the default image name here chosen by docker-compose, it might be better to specify an image name explicitly in the compose YAML and use it here.

Reply:
Yes, that is probably a good idea. I'll look into that shortly.